src/dir.targets

```diff
@@ -9,15 +9,15 @@
 
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition="('$(NetCoreBuild)' != 'true' or '$(OsEnvironment)'!='Windows_NT') and Exists('$(MSBuildToolsPath)\Microsoft.CSharp.targets')"/>
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" Condition="'$(NetCoreBuild)' == 'true' and '$(OsEnvironment)'=='Windows_NT'"/>
-  <Import Condition="'$(IsTraversalBuild)'!='true' and '$(NetCoreBuild)' == 'true' and Exists('$(CompilerToolsDir)\..\build\Microsoft.Net.Compilers.props')" Project="$(CompilerToolsDir)\..\build\Microsoft.Net.Compilers.props"/>
+  <Import Condition="'$(IsTraversalBuild)'!='true' and Exists('$(CompilerToolsDir)\..\build\Microsoft.Net.Compilers.props')" Project="$(CompilerToolsDir)\..\build\Microsoft.Net.Compilers.props"/>
```

Comment:
My pending PR #341 will remove this line (and it's equivalent in samples/dir.targets). Will this break things for !Windows?